Neos.Flow/Classes/Mvc/Routing/Dto/RouteLifetime.php

```diff
@@ -0,0 +1,88 @@
+<?php
+namespace Neos\Flow\Mvc\Routing\Dto;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * RouteLifetime to be associated with matched/resolved routes
+ *
+ * RouteLifetime can be set by Route Part handlers via the ResolveResult/MatchResult return values
+ * The lifetime will be set for the corresponding cache entries. The lowest lifetime is used if multiple are defined.
+ *
+ * @Flow\Proxy(false)
+ */
+final class RouteLifetime
+{
+
+    /**
+     * Lifetime value null = undefined, 0 = infinite, other values lifetime in s
+     * @var int|null
+     */
+    private $value;
+
+    /**
+     * @param int $value the lifetime
+     */
+    private function __construct(?int $value)
+    {
+        $this->value = $value;
+    }
+
+    /**
+     * Creates an empty instance without a specific lifetime
+     *
+     * @return RouteLifetime
+     */
+    public static function createUndefined(): self
+    {
+        return new static(null);
+    }
+
+    /**
+     * Creates an instance with a specific lifetime
+     *
+     * @param int $value Tag value satisfying the PATTERN_TAG regex
+     * @return RouteLifetime
+     */
+    public static function createFromInt(int $value): self
+    {
+        if ($value < 0) {
+            throw new \InvalidArgumentException(sprintf('RouteLifetime have has be non-negative integer value, %s given', $value), 1665928137);
+        }
+        return new static($value);
+    }
+
+    /**
+     * Merges two instances of this class combining results ny using the lowest lifetime
+     *
+     * @param RouteLifetime $lifetime
+     * @return RouteLifetime
+     */
+    public function merge(RouteLifetime $lifetime): self
+    {
+        $values = array_values(array_filter([$this->getValue(), $lifetime->getValue()]));
+        if (!empty($values)) {
+            return self::createFromInt(min($values));
```

Comment:
A lifetime of 0 (infinite) would always win the race, which isn't correct I suppose.
I would even suggest to make `0` a more explicit case (e.g. by adding `createInfinite()` and `isInfinite()`)

Reply:
adjusted the merge and added the methods `createInfinite` `isInfinite` and `isUndefined`

Reply:
Also i now only ignore the configuration if it is null for Lifetime or empty for tags. Other type issues would be raised by the php types in the dtos.